components/landing/navigation.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from "@/components/ui/button";
+import { Shell } from "lucide-react";
+
+// TODO: Add better responsiveness and mobile design, refactor Button into the button component
+
+export const Navigation = () => {
+    return (
+        <nav className='sticky top-0 z-[1] flex h-16 items-center bg-[#181818]'>
+            <a href='#landing'>
+                <Shell className='mx-5 h-full w-10 text-green-500'></Shell>
```

Comment:
Another component `Shell` that has no children. This can be self closing `<Shell className='...' />